docs/manual/src/docs/asciidoc/_includes/servlet/additional-topics/oauth2.adoc

```diff
@@ -96,6 +96,9 @@ The following sections go into more detail on each of the configuration options
 By default, the OAuth 2.0 Login Page is auto-generated by the `DefaultLoginPageGeneratingFilter`.
 The default login page shows each configured OAuth Client with its `ClientRegistration.clientName` as a link, which is capable of initiating the Authorization Request (or OAuth 2.0 Login).
 
+[NOTE]
+In order for `DefaultLoginPageGeneratingFilter` to show links for configured OAuth Clients, the registered `ClientRegistrationRepository` needs to also implement `Iterable`.
```

Comment:
I know it should be obvious but I think being explicit can't hurt...so `Iterable` -> `Iterable<ClientRegistration>`.

Also can we reference...see `InMemoryClientRegistrationRepository`